tomcat/src/test/resources/structure.yaml

```diff
@@ -0,0 +1,31 @@
+schemaVersion: 1.0.0
+
+commandTests:
+- name: 'CATALINA_HOME is set'
+  command: ['env']
+  expectedOutput: ['CATALINA_HOME=${tomcat.home}']
+- name: 'CATALINA_BASE is set'
+  command: ['env']
+  expectedOutput: ['CATALINA_BASE=${tomcat.base}']
+
+fileExistenceTests:
+- name: 'Tomcat home exists'
+  path: '${tomcat.home}'
+  isDirectory: true
+  shouldExist: true
+- name: 'Catalina script exists'
+  path: '${tomcat.home}/bin/catalina.sh'
+  isDirectory: false
+  shouldExist: true
+- name: 'Tomcat base exists'
+  path: '${tomcat.base}'
+  isDirectory: true
+  shouldExist: true
+- name: 'Webapps directory exists'
+  path: '${tomcat.base}/webapps'
+  isDirectory: true
+  shouldExist: true
```

Comment:
Maybe also add tests for `$CATALINA_BASE/temp` and `$CATALINA_BASE/conf/catalina.properties`?

And check working directory.